frontend/src/components/FeedListItem/FeedListItem.tsx

```diff
@@ -0,0 +1,34 @@
+import styles from "./FeedListItem.module.css";
+
+type FeedListItemProps = {
+  post: {
```

Comment:
nit: In typescript, is it possible to create reusable types (eg. can we create a type called "FeedPost" and nest that type inside FeedListItemProps?) If possible, I would suggest doing that since we will probably have to reuse the FeedPost types in other components

Reply:
I'm not aware if something like this exists. I will look into it